.gitignore

```diff
@@ -43,4 +43,3 @@ yarn-debug.log*
 
 
 # custom
-Gemfile.lock
```

Comment:
This change is unintentional, correct?
Let's talk about why this happens tomorrow to avoid recurrence

Reply:
I was advised to remove this by a ta so it was intentional but I didn't fully understand why... I've just looked into it a bit and apparently without Gemfile.lock  its possible to run into issues where the production server loads a newer version of a gem than you have locally and breaks the application. Some people seem to disagree but i found out that the devise gem actually caused problems this way when it was updated to 3.1 so maybe safest not to add it to ignore?